public/js/app/shared/templates/ysearch.html

```diff
@@ -9,16 +9,20 @@
         <div class="y-search-container" ng-show="search.showSearchResults && search.results.length > 0">
             <a class="form-control y-search-results" ng-click="hideSearchResults()"
                ui-sref="base.product.detail( {productId: result.objectID} )" ng-repeat="result in search.results">
-                <div class="attribute">
-                <span class="y-search-result-name text-left"
-                      ng-bind-html="result._highlightResult.name[0].value"></span>
+                <div class="y-search-result-thumbnail">
+                    <img class="y-search-result-image" ng-src="{{result.image}}" />
+                </div>
+                <div class="y-search-result-attributes">
+                    <div class="y-search-result-name text-left"
+                          ng-bind-html="result._highlightResult.name[0].value"></div>
+                    <div class="y-search-result-category text-left">{{'IN' | translate}} {{result.categories.join(', ')}}</div>
+                    <div class="y-search-result-price text-left">{{result.prices[currency] | currency: currencySymbol }}</div>
```

Comment:
Cool that is working again.
No, ysearchController won't be reloaded when site is changed. You are right, one way would be detecting that site is changed and then update currency. But that will probably require a bit more logic. We do not have event when site is changed at the moment.

So what you can also do at this moment is adding code that is getting currency in the method showSearchResults so it will look something like this.
```
scope.showSearchResults = function () {

            scope.currency = GlobalData.getCurrency();
            scope.currencySymbol = GlobalData.getCurrencySymbol();

            scope.search.showSearchResults = true;
            if (scope.search.text !== '') {
                if (scope.search.results.length === 0) {
                    scope.doSearch(scope.search.text, 0);
                }
            }
        };
```
Then every time search is opened you will get latest currency that is selected and it will be always correct. 
You can see two different sites with different currencies here: [shops.yaas.io/defaultproj](shops.yaas.io/defaultproj) . Click on the globe in the top right corner to see site selector.

Also please fix 3 failing tests. You can run tests from /script folder. There is a file called test.sh that is used to run unit tests.